syslog/modules/lpeg/sendmail.lua

```diff
@@ -0,0 +1,30 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+# Sendmail module
+
+## Variables
+### LPEG Grammars
+* `grammar`
+--]]
+
+local l = require "lpeg"
+l.locale(l) -- we want l.alpha etc...
+
+local rawset = rawset
+
+local M = {}
+setfenv(1, M) -- Remove external access to contain everything in the module
+
+local sep = l.P", "
+local key = l.C(l.alpha^1)
+local value = l.C((l.P(1) - sep)^1)
+local pair = l.Cg(key * "=" * value) * sep^-1
+
+local sendmailid = l.Cg(l.alnum^-8 * l.digit^6, "sendmailid") * l.P": "
```

Comment:
I believe this is too restrictive e.g. ` e6FFrpW22493:` Since it doesn't have a defined format or length:
- leave it alnum like it was and require at least one pair (as I haven't see an error lines with that format format)
- just check that it ends in a digit followed by a colon

Reply:
you are correct (ignore the comment above): https://fossies.org/dox/sendmail.8.15.2/queue_8c_source.html#l05381